frontend/src/lib/ui/icons/TriangleAlertIcon.tsx

```diff
@@ -1,7 +1,13 @@
-export const TriangleAlertIcon = () => (
+export const TriangleAlertIcon = ({
+  width = '1em',
+  height = '0.84em',
+}: {
+  width?: number | string;
+  height?: number | string;
+}) => (
   <svg
-    width="1em"
```

Comment:
That will break other usage. Prefer to use `fontSize` to define icon size. E.g. here width is `1em`, if you want the width of an icon to be 20px use `font-size: 2px`